backport/backport-languages.py

```diff
@@ -112,7 +112,7 @@ def main():
 
     for language_file in language_files:
         print("Backporting %s ..." % language_file[len("src/lang/") :])
-        backport_language(language_file, blacklisted_ids, diff_to_stdout=args.diff)
+        backport_language(language_file.replace('\\', '/'), blacklisted_ids, diff_to_stdout=args.diff)
```

Comment:
I wonder if we should do it only when ran on Windows. As this normally ofc isn't an issue .. Something like `sys.platform == "win32"` or something.

Or you know .. use a sane OS! :P :D